android/app/build.gradle

```diff
@@ -110,6 +114,26 @@ android {
         versionName "1.3.46-1"
     }
 
+    flavorDimensions "default"
+    productFlavors {
+        production {
+        }
```

Comment:
Are we just leaving this empty object to make it clearer to other devs that it COULD be used? if so can you please just add a small comment inside to explain? Thanks!

Reply:
I did some cleanup here.